test/helper.lua

```diff
@@ -230,4 +237,77 @@ function helpers.get_other_storage_bucket_id(cluster, bucket_id)
     ]], {bucket_id})
 end
 
+-- Determine a replicaset alias by a server object.
+function helpers.get_replicaset_alias(cluster, needle_server)
+    for _, replicaset in ipairs(cluster.replicasets) do
+        for _, server in ipairs(replicaset.servers) do
+            if server.alias == needle_server.alias then
+                return replicaset.alias
+            end
+        end
+    end
+    error(('Unable to find replicaset alias for server %s'):format(
+        needle_server.alias))
+end
+
+-- Accumulate statistics from given storages.
+--
+-- The statistics is grouped by replicasets.
+--
+-- Example of a return value:
+--
+-- {
+--     ['s-1'] = {
+--         select_requests = 1,
+--     },
+--     ['s-2'] = {
+--         select_requests = 0,
+--     },
+-- }
+function helpers.collect_storage_stat(cluster, aliases)
+    local res = {}
+
+    helpers.call_on_servers(cluster, aliases, function(server)
+        -- Collect the statistics.
+        local storage_stat = server.net_box:call('storage_stat')
+
+        -- Initialize if needed.
+        local replicaset_alias = helpers.get_replicaset_alias(cluster, server)
+        if res[replicaset_alias] == nil then
+            res[replicaset_alias] = {}
+        end
+
+        -- Accumulate the collected statistics.
+        for key, val in pairs(storage_stat) do
+            local old = res[replicaset_alias][key] or 0
+            res[replicaset_alias][key] = old + val
+        end
+    end)
+
+    return res
+end
+
+-- Difference between 'a' and 'b' storage statistics.
```

Comment:
`a` and `b` are replicasets, not storages. If I got it right.

Reply:
`a` and `b` are statistics objects, where each contains statistics for several replicasets (each replicaset entry may contain several metrics, but only 'select_requests' for now). I think I can name an object of this kind just 'storage statistics'.

The example (it is referenced just below in this comment) describes the return value structure and it should be enough to make it clear how to use the object. At least it looks more clear for me than the description I wrote here :)